byoqm/runner/runner.py

```diff
@@ -102,13 +103,15 @@ def _run_metrics(self) -> Dict:
 
     def _generate_violations_table(self, results: Dict, time: str):
         list_of_violations = []
-        for _, violations in results.items():
-            if type(violations) is list:
-                list_of_violations.extend(violations)
+        for _, result in results.items():
+            # need a better name than result...
+            if type(result) is not Result:
+                continue
```

Comment:
When `result` not be a `Result` type?

Reply:
The dictionary given can also return integers from aggregations